sorc/chgres_cube.fd/wam_climo_data.f90

```diff
@@ -0,0 +1,2758 @@
+!! output variables:                                                 
+!! @param [out] d density array with length of 9
+!! @param [out] t temperature array with length of 2
+!!              d array contains:
+!!               (1) - he number density(cm-3)                                 
+!!               (2) - o number density(cm-3)                                  
+!!               (3) - n2 number density(cm-3)                                 
+!!               (4) - o2 number density(cm-3)                                 
+!!               (5) - ar number density(cm-3)                                 
+!!               (6) - total mass density(gm/cm3) [includes anomalous oxygen]  
+!!               (7) - h number density(cm-3)                                  
+!!               (8) - n number density(cm-3)                                  
+!!               (9) - anomalous oxygen number density(cm-3)                   
+!!              t array conyains:
+!!               (1) - exospheric temperature                                  
+!!               (2) - temperature at alt                                      
+!!                                                                      
+!! @author Hann-Ming Henry Juang
+      subroutine gtd7d(iyd,sec,alt,glat,glong,stl,f107a,f107,ap,mass,d,t)
+      use wam_gtd7bk_mod, only: imr
+!x----common/metsel/imr 
+      dimension d(9),t(2),ap(7),ds(9),ts(2) 
+      call gtd7(iyd,sec,alt,glat,glong,stl,f107a,f107,ap,mass,d,t) 
+!       total mass density                                              
+!                                                                       
+        if(mass.eq.48) then 
+         d(6) = 1.66e-24*(4.*d(1)+16.*d(2)+28.*d(3)+32.*d(4)+40.*d(5)+  &
+             d(7)+14.*d(8)+16.*d(9))                                    
+         if(imr.eq.1) d(6)=d(6)/1000. 
+         endif 
+      return 
+      end subroutine gtd7d
+
+!-----------------------------------------------------------------------
+!> Find altitude of pressure surface (press) from gtd7             
+!!
+!!    input:                                                            
+!! @param [in] iyd - year and day as yyddd                                    
+!! @param [in] sec - ut(sec)                                                  
+!! @param [in] glat - geodetic latitude(deg)                                  
+!! @param [in] glong - geodetic longitude(deg)                                
+!! @param [in] stl - local apparent solar time(hrs)                           
+!! @param [in] f107a - 3 month average of f10.7 flux                          
+!! @param [in] f107 - daily f10.7 flux for previous day                       
+!! @param [in] ap - magnetic index(daily) or when sw(9)=-1. :                 
+!!                - array containing:                                         
+!!                  (1) daily ap                                              
```

Comment:
In order to make this a numbered list, use '1.' instead of '(1)'. So like this instead:
```
!!                  1. daily ap 
!!                  2. 3 hr ap index for current time 
!!                  3. 3 hr ap index for 6 hrs before current time
```